client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,86 @@
+import React , {useCallback, useState} from 'react';
+import {Button, Card, InputGroup, FormControl} from 'react-bootstrap';
+import {setCalibration as sendCalibrationValue, resetCalibration} from 'api/common/powerModel';
+// import {useSensorData} from 'api/v2/sensors/data';
+
+/**
+ * Boost distance claibration input form
+ * 
+ * @returns {React.Component} Component
+ */
+function BoostCalibrationInput() {
+    const [calibrationValue, setCalib] = useState('');
+
+    const handleCalibrationChange = useCallback(
+        (event) => {
+        setCalib(event.target.value);
+        },
+        [setCalib],
+    );
+
+    const onSet = useCallback(
+        () => {
+          const calibValue: number = parseInt(calibrationValue, 10);
+          if (Number.isNaN(calibValue)) {
+            alert("Please enter an integer value only!");
+          }
+          else {
+            console.log(calibValue);
```

Comment:
`console.log` is great for debugging but ideally we won't have any in production!